src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const newObject = {};
+
+  const uniqueValues = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
+    if (uniqueValues.hasOwnProperty(robotValues[i])) {
+      return null;
+    }
+
+    uniqueValues[robotValues[i]] = true;
+  }
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    newObject[value] = key;
```

Comment:
Just check here if value is not present in new robot

Reply:
Thank You. I think I got it right. Check please